101-route-server-quagga/azuredeploy.json

```diff
@@ -0,0 +1,467 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
```

Comment:
add a defaultValue [resourceGroup().location] and remove the param value from the param file

Reply:
done